demo/privileged/auto-unpriv/index.html

```diff
@@ -95,125 +95,157 @@
-                  <b>Send via P2P channel</b> <span class="glyphicon glyphicon-transfer"></span>
-                </label>
+              <div id="selected_users_panel">
+                <h5>Targeted peers to message / file transfer to</h5>
+                <p>Peers: <span class="all">All</span> <span class="selected-users"></span></p>
+                <p><button id="clear-selected-users" class="btn btn-primary btn-rounded">Reset Targeted peers</button></p>
+              </div>
+              <div id="file_panel">
+                <br>
+                <div class="input-group">
+                  <input type="file" class="form-control" title="Select a file to upload" id="file_input">
+                  <span class="input-group-btn">
+                    <button id="send_file_btn" class="form-control" title="Upload file" type="button">
+                      <span class="glyphicon glyphicon-cloud-upload"></span> <b>Upload file</b>
+                    </button>
+                  </span>
+                </div>
+              </div>
+              <div id="dataURL_panel">
+                <br>
+                <div class="input-group">
+                  <input type="file" class="form-control" title="Select a file to upload" id="dataURL_input">
+                  <span class="input-group-btn">
+                    <button id="send_dataURL_btn" class="btn btn-default" title="Send Image" type="button">
+                      <span class="glyphicon glyphicon-cloud-upload"></span> <b>Send Image</b>
+                    </button>
+                  </span>
+                </div>
               </div>
-            </form>
-            <textarea id="chat_input" placeholder="Enter your chat message here" class="well" disabled></textarea>
+              <div id="chat_panel">
+                <br>
+                <hr/>
+                <div>
+                  <h2>Chat Log</h2>
+                </div>
+                <div class="panel panel-default panel-chat">
+                  <div id="chat_body" class="panel-body">
+                    <div id="chat_log" class="list-group"></div>
+                  </div>
+                </div>
+                <form role="form">
+                  <div class="checkbox">
+                    <label>
+                      <input id="send_data_channel" type="checkbox">
+                      <b>Send via P2P channel</b> <span class="glyphicon glyphicon-transfer"></span>
+                    </label>
+                  </div>
+                </form>
+                <textarea id="chat_input" placeholder="Enter your chat message here" class="well panel-chat" disabled></textarea>
+                <br>
+              </div>
+            </div>
           </div>
         </div>
       </div>
+    </section>
+    <div class="">
+      <div class="container">
+        <p class="text-center">
+          &copy; Temasys Communications Pte Ltd 2016. All Rights Reserved.
```

Comment:
Can be changed to Temasys Communications Pte Ltd **2018** wherever you encounter

Reply:
Sure